server/server.js

```diff
@@ -1,5 +1,6 @@
 // retrieve configuration from main config file
 const config = require('./config/base.config.json');
+config.mongoose = require('./config/mongoose.config.json');
```

Comment:
@BernardJeremy @EIP-SAM/development 
I'm making a remark here, but it is applicable to some other PRs, should we separate the configuration files, like it has be done here, or should we merge them all in a solo config file ?
I'm first asking to Jeremy but any answer is good to hear.